tools/calyx-pass/test.futil

```diff
@@ -0,0 +1,31 @@
+import "primitives/core.futil";
```

Comment:
How is this test tested?

Reply:
It's not, I'm going to remove it

Reply:
I've moved this to an examples directory with a command explaining how to use it.